src/irmin-test/common.ml

```diff
@@ -220,13 +220,13 @@ module Make_helpers (S : Generic_key) = struct
         let module Conf = Irmin.Backend.Conf in
         let generate_random_root config =
           let id = Random.int 100 |> string_of_int in
-          let root_value =
-            match Conf.find_root config with
-            | None -> "test_" ^ id
-            | Some v -> v ^ "_" ^ id
```

Comment:
Was this case unused?

Reply:
Agree! But my question was with respect to ignoring the pre-configured root value `v` and always using `"test_"` in your revision: was this actually unused? (in which case your simplification makes sense :) )

Reply:
My bad! It was used, I reverted back to using it!